### PR TITLE
[Pizza Test] Creating new migration to populate pizza test

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This PR is introducing the migration that populates the Pizza Test. This migration is populating the Overview (before the test starts), the background information (6 sections) and the questions (10 emails).

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Overview (before the test starts):**
image

**Background Informations (EN):**
image

**Background Informations (FR):**
image

**Questions/Emails (EN):**
image

**Questions/Emails (FR):**
image

**Tree-View - Organizational Structure (EN):**
image

**Tree-View - Organizational Structure (FR):**
image

**Tree-View - Team Information (EN):**
image

**Tree-View - Team Information (EN):**
image

# Testing

Manual steps to reproduce this functionality:

1.  _docker-compose down_.
2.  _docker-compose up_.
3.  Open the following APIs:
- http://localhost/api/test-questions/?test_name=emibPizzaTest
- http://localhost/api/test-meta-data/?test_name=emibPizzaTest
4.  Make sure that you get the test's content (overview, background info and questions/emails).

# Checklist

Applicable for all code changes.

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] ~~I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [ ] ~~My changes look good on IE 11+ and Chrome~~

### CODE REVIEW

**Comment by caleybrock** [comment]
I don't see the org charts / address books in your screenshots - will they be added with this PR?

**Comment by fnormand01** [comment]
@caleybrock I'll add the needed tree-views in this PR.

File: backend/custom_models/migrations/0018_upload_pizza_test.py [hunk 1]
```diff
@@ -0,0 +1,2093 @@
...
+        .last()
+    )
+    l_french = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="fr", ISO_Code_2="fr-ca")
+        .last()
+    )
+    pizza_test_item_id = (
+        item_text.objects.using(db_alias)
+        .filter(text_detail="Pizza Test", language=l_english)
```
Comment (MichaelCherry) [inline backend/custom_models/migrations/0018_upload_pizza_test.py]: As discussed with Caley offline, we need to shift from looking up via text detail to using the test name. I will be changing this for the eMiB globally in a PR in the near future. I'll add a comment here when I do so you can do the same thing for the pizza test

File: backend/custom_models/migrations/0018_upload_pizza_test.py [hunk 2]
```diff
@@ -0,0 +1,2093 @@
...
+### Processing grant applications
+
+The Rebel Team grants funds for pizzaism activities, products and services in the Rebel region that meet the funtimes criteria established by the Saucy Research Team for each funtimes program.
+
+#### The Review Process
+
+Funtimes decisions are based on a rigorous review process involving the following steps:
+
+1. **Initial slicing up of applications.** Funtimes support assistants slice applications using an internal cheese management software to ensure that there is no missing toppings. The software automatically assigns applications to the analyst responsible for the primary targeted pizzaismsector.
+2. **Analysis of the applications.** Analysts assign ratings for each of the criteria used to review grant applications. Funtimes criteria typically include the potential economic, social, cultural, and culinary impacts, as well as the efficiency of the proposed use
```
Comment (MichaelCherry) [inline backend/custom_models/migrations/0018_upload_pizza_test.py]: See earlier comment

File: backend/custom_models/migrations/0018_upload_pizza_test.py (hunk 1 above)
Comment (MichaelCherry) [reply backend/custom_models/migrations/0018_upload_pizza_test.py]: See PR #446

File: backend/custom_models/migrations/0018_upload_pizza_test.py (hunk 2 above)
Comment (MichaelCherry) [reply backend/custom_models/migrations/0018_upload_pizza_test.py]: See PR #446

**Comment by fnormand01** [comment]
Tree-views added.
See commit #https://github.com/code-for-canada/project-thundercat/pull/445/commits/c6129264976ff40081a88c63e60fc79ce3965a1e and commit #https://github.com/code-for-canada/project-thundercat/pull/445/commits/1e66dc48aa2e9f2686403db075235041ae8cafa8

File: backend/custom_models/migrations/0018_upload_pizza_test.py (hunk 1 above)
Comment (fnormand01) [reply backend/custom_models/migrations/0018_upload_pizza_test.py]: Updated in commit #https://github.com/code-for-canada/project-thundercat/pull/445/commits/1e786ad66593d49ddc46e3b745f54287b22f7ec8

File: backend/custom_models/migrations/0018_upload_pizza_test.py (hunk 2 above)
Comment (fnormand01) [reply backend/custom_models/migrations/0018_upload_pizza_test.py]: I'm not sure to understand your previous comment with this section of the code?

Comment (MichaelCherry) [reply backend/custom_models/migrations/0018_upload_pizza_test.py]: woops the line number changed. I'll comment again if you didn't fix it

**Comment by fnormand01** [comment]
GitHub is not displaying the test for some reason.
However, all tests passed. Here is a screenshot of the passing tests:

image

**Pull request merged**